### PR TITLE
Make node

### PR DESCRIPTION
**Pull request by atjn**
Node go brrrr

### CODE REVIEW

File: src/main/java/dk/aau/p4/abaaja/Lib/Nodes/BlockNode.java [hunk 1]
```diff
@@ -1,5 +1,22 @@
 package dk.aau.p4.abaaja.Lib.Nodes;
 
+import dk.aau.p4.abaaja.Lib.Visitors.INodeVisitor;
+
+import javax.sound.sampled.Line;
```
Comment (TSKsmiley) [inline src/main/java/dk/aau/p4/abaaja/Lib/Nodes/BlockNode.java:5]: uh is this correct

**Review by TSKsmiley — CHANGES_REQUESTED** [review]

File: src/main/java/dk/aau/p4/abaaja/Lib/Nodes/IfStateNode.java [hunk 2]
```diff
@@ -1,5 +1,34 @@
 package dk.aau.p4.abaaja.Lib.Nodes;
 
+import dk.aau.p4.abaaja.Lib.Visitors.INodeVisitor;
+
+
+import javax.swing.plaf.nimbus.State;
```
Comment (atjn) [inline src/main/java/dk/aau/p4/abaaja/Lib/Nodes/IfStateNode.java:6]:
```suggestion
```
Bing bong?

File: src/main/java/dk/aau/p4/abaaja/Lib/Nodes/BlockNode.java (hunk 1 above)
Comment (BaBrixx) [reply src/main/java/dk/aau/p4/abaaja/Lib/Nodes/BlockNode.java:5]:
```suggestion
import dk.aau.p4.abaaja.Lib.Nodes.LineNode;
```

Comment (BaBrixx) [reply src/main/java/dk/aau/p4/abaaja/Lib/Nodes/BlockNode.java:5]:
```suggestion
```

**Review by BaBrixx — CHANGES_REQUESTED** [review]

**Review by TSKsmiley — APPROVED** [review]
:D

**Review by BaBrixx — APPROVED** [review]
LGTM

**Pull request merged**